src/mon/OSDMonitor.cc

```diff
@@ -641,7 +641,10 @@ class OSDUtilizationDumper : public CrushTreeDumper::Dumper<F> {
     int64_t kb = 0, kb_used = 0, kb_avail = 0;
     double util = 0;
     if (get_bucket_utilization(qi.id, &kb, &kb_used, &kb_avail))
+    if (kb_used) {
+      assert(kb > 0);
```

Comment:
this does not fix the divide by zero issue.  if kb is 0, monitor crashes as before.